### ISSUE TITLE
get Woofmark tokenizer to run all the time, instead of only on demarkdowning

### ISSUE THREAD

**Issue by jywarren** [publiclab/PublicLab.Editor#8]

**Comment by rexagod** [comment]
@jywarren Do you want to incorporate this in markdown as well? Is that what you're saying here?

````js
tokenizers: [
          {
            token: /(^|\s)@([A-z\_]+)\b/g, // @callouts
            transform: function (all, separator, id) {
              return separator + '<a href="/profile/' + id + '">@' + id + '</a>';
            }
          },
          {
            token: /(^|\s)#([A-z\-]+)\b/g, // #hashtags
            transform: function (all, separator, id) {
              return separator + '<a href="/tag/' + id + '">#' + id + '</a>';
            }
          },
          {
            token: /\[([\S\:\/]+)\]/, // [power:tags]
            transform: function (all, keywords) {
              return '<div class="powertags">Power tag: ' + keywords + '</div>';
            }
          }
        ]
````

**Comment by jywarren** [comment]
I think originally this was because as you typed `@warren` for example, it would only convert it into a link if you changed the mode, and it ran its filters. This still seems to be the case! Could we linkify automatically perhaps after the atwho runs?

**Comment by Shulammite-Aso** [comment]
> I think originally this was because as you typed `@warren` for example, it would only convert it into a link if you changed the mode, and it ran its filters. This still seems to be the case! Could we linkify automatically perhaps after the atwho runs?

@jywarren @sagarpreet-chadha  for this, i just figured out we don't currently have the Atwho.js Data Module?
There's a documentation about how to include it here in the [readme](https://github.com/publiclab/PublicLab.Editor#atwhojs-data-module). i can work on this, if we need to.

Thanks!!!

**Comment by sagarpreet-chadha** [comment]
I think this feature was removed, @jywarren were you discussing about this feature in the Staff meeting?
I also do not have the full context here.

**Comment by jywarren** [comment]
Oh my gosh, getting Atwho.js working here would be AMAZING. If it's not too much of a detour in your work, we'd really appreciate it!

**Comment by Shulammite-Aso** [comment]
> Oh my gosh, getting Atwho.js working here would be AMAZING. If it's not too much of a detour in your work, we'd really appreciate it!


I think i should do this then.

**Comment by Shulammite-Aso** [comment]
@jywarren @sagarpreet-chadha this will have to be done at plots2.  The editor example has this setup in the [data](https://github.com/publiclab/PublicLab.Editor/tree/main/examples/data) directory already, and is working there.

Will open an issue for this at plots2 then work on it.

**Comment by Shulammite-Aso** [comment]
Or maybe transfer this one?

**Comment by sagarpreet-chadha** [comment]
Yes let's make a new issue there and linking this issue. Thanks 😄
